src/main/resources/static/js/js_utils.js

```diff
@@ -408,7 +408,11 @@ function loadUserFormSettings(formElement){
     } else {
         let savings = JSON.parse(localStorage.getItem("savedUserSettings"));
         allCheckboxes.forEach(checkbox => {
-            checkbox.checked = savings[checkbox.id];
+            if(undefined === savings[checkbox.id]) {
+                checkbox.checked = true;
```

Comment:
i would suggest that when there is no saving it should get the checked state from the server request.
(let the check state be the state it is / which got set by thymeleaf)